server/routerlicious/packages/routerlicious-base/src/test/alfred/io.spec.ts

```diff
@@ -63,7 +63,7 @@ if (!Lumberjack.isSetupCompleted()) {
 }
 
 describe("Routerlicious", () => {
-	describe("Alfred", () => {
+	describe("Nexus", () => {
```

Comment:
If this file is purely websocket-related, maybe move it to a nexus folder instead of an alfred one? Or if the tests are mixed, extract the websockets ones to a new file in a nexus folder.